src/UnderlineNav.tsx

```diff
@@ -111,4 +111,7 @@ const UnderlineNavLink = styled.a.attrs<StyledUnderlineNavLinkProps>(props => ({
 UnderlineNavLink.displayName = 'UnderlineNav.Link'
 
 export type UnderlineNavLinkProps = ComponentProps<typeof UnderlineNavLink>
+/**
+ * @deprecated UnderlineNav to be deprecated in the next major Primer React release. Please use the new responsive and accessible UnderlineNav. See https://primer.style/react/drafts/UnderlineNav2 for more details.
```

Comment:
Slight wording suggestion:

```suggestion
 * @deprecated UnderlineNav is deprecated and will be replaced by the draft `UnderlineNav` in the next major release. See https://primer.style/react/drafts/UnderlineNav2 for more details.
```

The switch to _is deprecated_ is to match using `@deprecated` which is saying that this item is currently deprecated.

Unfortunately, I feel like the tenses of "will be deprecated" and "is deprecated" are challenging with our current setup (the different entrypoints). Ideally with deprecation warnings, we should be saying:

1. This feature is deprecated and will be changed in the next major release
2. Suggest an alternative to use to make updating easier for the next major release

Reply:
Thanks for the suggestion! It makes sense

>Unfortunately, I feel like the tenses of "will be deprecated" and "is deprecated" are challenging with our current setup (the different entrypoints).

I know 😭 I re-thought about the wording and updated the deprecated docs https://github.com/primer/react/pull/2733 would love to hear what you think 💖